content/developer/howtos.rst

```diff
@@ -10,6 +10,9 @@ How-to guides
 
     howtos/scss_tips
     howtos/web_services
```

Comment:
I think the order of the cards is good as it is, but then `web_services` should be moved after `javascript_*` in the toctree.